assumerole.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+  external_id = coalesce(var.assumerole_role_external_id, random_password.external_id.result)
```

Comment:
Why? Why give a random password the ability to assume admin access to your parent account?

Reply:
The parent account de facto has such access anyway. This is just explicitly connecting the account intended for terraform usage to the subset of users that we decide should be able to run terraform in the subaccounts.

The external ID is a recommended best practice. It doesn't confer access to anything all by itself. It does the opposite, actually.. if you're granted access to assume this role in the subaccount, you also need to "know" the external ID to use it.